webpack.config.js

```diff
@@ -86,7 +86,9 @@ module.exports = {
             directory: path.join(__dirname, "static"),
         },
         compress: true,
-        historyApiFallback: true,
+        historyApiFallback: {
+            disableDotRule: true
```

Comment:
add a comment as to why this has changed

Reply:
Done, some test data I used had individual IDs with dots, this rule prevented the use of these IDs as url parameters.